### PR TITLE
Add CrUX INP query

### PR DESCRIPTION
**Pull request by tunetheweb**
Makes progress on https://github.com/HTTPArchive/httparchive.org/issues/595

### CODE REVIEW

File: sql/histograms/cruxInp.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#standardSQL
+CREATE TEMPORARY FUNCTION spreadBins(bins ARRAY<STRUCT<start INT64, `end` INT64, density FLOAT64>>)
```
Comment (rviscomi) [inline sql/histograms/cruxInp.sql:2]: All INP bins should be 25ms wide, so we can remove this.

Still needed for other metrics that have older data before bin spreading was built-in.

**Review by rviscomi — CHANGES_REQUESTED** [review]

File: sql/histograms/cruxInp.sql (hunk 1 above)
Comment (tunetheweb) [reply sql/histograms/cruxInp.sql]: Done!

File: sql/timeseries/cruxFastInp.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  IF(device = 'desktop', 'desktop', 'mobile') AS client,
+  SAFE_DIVIDE(
+    COUNT(DISTINCT IF(IS_GOOD(fast_inp, avg_inp, slow_inp), origin, NULL)),
+    COUNT(DISTINCT IF(IS_NON_ZERO(fast_inp, avg_inp, slow_inp), origin, NULL))
+  ) * 100 AS percent
+FROM
+  `chrome-ux-report.materialized.device_summary`
+WHERE
+  device IN ('desktop', 'phone') AND
+  yyyymm >= 202204
```
Comment (rviscomi) [inline sql/timeseries/cruxFastInp.sql:24]: Data exists for the previous `responsiveness` metric under this field name back to 202202. Slightly different implementation but still good to track historical progress.
```suggestion
  yyyymm >= 202202
```

File: sql/timeseries/cruxSlowInp.sql [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  IF(device = 'desktop', 'desktop', 'mobile') AS client,
+  SAFE_DIVIDE(
+    COUNT(DISTINCT IF(IS_POOR(fast_inp, avg_inp, slow_inp), origin, NULL)),
+    COUNT(DISTINCT IF(IS_NON_ZERO(fast_inp, avg_inp, slow_inp), origin, NULL))
+  ) * 100 AS percent
+FROM
+  `chrome-ux-report.materialized.device_summary`
+WHERE
+  device IN ('desktop', 'phone') AND
+  yyyymm >= 202204
```
Comment (rviscomi) [inline sql/timeseries/cruxSlowInp.sql:24]:
```suggestion
  yyyymm >= 202202
```

**Review by rviscomi — APPROVED** [review]

File: sql/timeseries/cruxFastInp.sql (hunk 2 above)
Comment (tunetheweb) [reply sql/timeseries/cruxFastInp.sql]: Might also give us something more interesting to graph!

**Pull request merged**